spouse/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Stored data
+*.pkl
+trained_spouse_model
```

Comment:
nit: is there a way to make this consistent with the other checkpoints of other notebooks? I believe there's a `.gitignore` for `checkpoints` in master: https://github.com/snorkel-team/snorkel-tutorials/blob/master/.gitignore#L137